### DISCUSSION TITLE
Windows Installer (MSI)

### DISCUSSION THREAD

**Discussion by ldlx** [WinMerge/winmerge#2530] [Ideas]
Hello,
I created a MSI for WinMerge a while ago - are you interested in using/publishing this?
Kind regards

**Comment by sdottaka** [comment]
For now, I have no plans to change my main installer creation tool from Inno Setup, but if the MSI installer you created is closer to the current one than the one below, I would appreciate it if you could make a pull request. However, whether it will actually be used depends on how complete it is.

https://github.com/WinMerge/winmerge/tree/master/Installer/WiX
